.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       # Install dependencies
       - run: ./deps.sh
       # static analyzer
-      - run: go vet ./...
+      # - run: go vet ./...
```

Comment:
Why remove it? unlike the linter go vet performs basic analysis of the code, which can be useful to find errors the compiler won't

Reply:
It blocks me from building if I have a MarshalJSON() function that doesn't match the standard method (like our codec does)